docs/apache-airflow/authoring-and-scheduling/datasets.rst

```diff
@@ -321,6 +321,34 @@ Example:
 Note that this example is using `(.values() | first | first) <https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.first>`_ to fetch the first of one dataset given to the DAG, and the first of one DatasetEvent for that dataset. An implementation can be quite complex if you have multiple datasets, potentially with multiple DatasetEvents.
 
 
+Manipulating queued dataset events through REST API
+---------------------------------------------------
+
+.. versionadded:: 2.9
+
+In this example, the DAG ``waiting_for_dataset_1_and_2`` will be triggered when tasks update both datasets "dataset-1" and "dataset-2". Once "dataset-1" is updated, Airflow creates a record. This ensures that Airflow knows to trigger the DAG when "dataset-2" is updated. We call such records queued dataset events.
+
+.. code-block:: python
+
+    with DAG(
+        dag_id="waiting_for_dataset_1_and_2",
+        schedule=[Dataset("dataset-1"), Dataset("dataset-2")],
+        ...,
+    ):
+        ...
```

Comment:
What is the added value of this paragraph?
It feels like repeating what is previously explain in Multiple Datasets sectionץ

I think the section should be just about the API endpoints (BTW is it just API or also CLI?)

Reply:
I'm trying to explain the term "queued dataset events" here. Under the hook, it's DatasetDagRunQueue, but ASAIK, we don't want to expose this internal thing